documentation-site/examples/input/password.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React from 'react';
+import {Button, KIND} from 'baseui/button';
+import {Input} from 'baseui/input';
+import ViewHide from 'baseui/icon/view-hide';
+import ViewShow from 'baseui/icon/view-show';
+
+export default () => {
+  const [value, setValue] = React.useState('1234');
+  const [hidden, setHidden] = React.useState(true);
+  const title = hidden
+    ? 'Show password text'
+    : 'Hide password text';
+  return (
+    <Input
+      value={value}
+      onChange={event => setValue(event.target.value)}
+      type={hidden ? 'password' : 'text'}
+      overrides={{
+        After: () => {
+          return (
+            <Button
+              kind={KIND.minimal}
+              title={title}
+              onClick={() => setHidden(h => !h)}
+            >
+              {hidden ? (
+                <ViewShow title={title} color="black" />
```

Comment:
<img width="678" alt="Screen Shot 2019-07-22 at 5 31 28 PM" src="https://user-images.githubusercontent.com/1387913/61673996-96704700-aca6-11e9-8653-7478a5bdee96.png">

The color should be dynamic.